substrate/frame/support/src/traits/tokens/fungible/union_of.rs

```diff
@@ -0,0 +1,1005 @@
+// This file is part of Substrate.
+
+// Copyright (Criterion) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Types to combine some `fungible::*` and `fungibles::*` implementations into one union
+//! `fungibles::*` implementation.
```

Comment:
What about combining some `fungible::*` and `fungible::*` into one union `fungibles::*`?

Not something we need to necessarily do in this PR, just wondering if it's been considered.

Reply:
usually `fungible::*` used for native currency, I do not see now examples where we have two impls for `fungible::*` in a single runtime. when we have it we can add